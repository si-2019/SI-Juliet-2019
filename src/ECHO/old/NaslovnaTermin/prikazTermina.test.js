import React from "react";
import PrikazTermina from "./prikazTermina";

import { shallow } from "enzyme";

it("renderuje formu za prikaz odredjenog dana", () => {
  const component = shallow(<PrikazTermina />);
  expect(component).toMatchSnapshot();
});
it("postoje li prikazi liste termina za ponedjeljak", () => {
  const wrapper = shallow(<PrikazTermina />);
  expect(wrapper.find("#Ponedjeljak").exists()).toBe(true);
});
it("postoje li prikazi liste termina za utorak", () => {
  const wrapper = shallow(<PrikazTermina />);
  expect(wrapper.find("#Utorak").exists()).toBe(true);
});
it("postoje li prikazi liste termina za srijedu", () => {
  const wrapper = shallow(<PrikazTermina />);
  expect(wrapper.find("#Srijeda").exists()).toBe(true);
});
it("postoje li prikazi liste termina za četvrtak", () => {
  const wrapper = shallow(<PrikazTermina />);
  expect(wrapper.find("#Cetvrtak").exists()).toBe(true);
});
it("postoje li prikazi liste termina za petak", () => {
  const wrapper = shallow(<PrikazTermina />);
  expect(wrapper.find("#Petak").exists()).toBe(true);
});
