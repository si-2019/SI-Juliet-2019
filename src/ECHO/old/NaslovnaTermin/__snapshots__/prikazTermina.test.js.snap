// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderuje formu za prikaz odredjenog dana 1`] = `ShallowWrapper {}`;
